cmd/soroban-cli/src/commands/contract/alias/show.rs

```diff
@@ -0,0 +1,62 @@
+use std::fmt::Debug;
+
+use clap::{command, Parser};
+
+use crate::commands::{config::network, global};
+use crate::config::locator;
+use crate::print::Print;
+
+#[derive(Parser, Debug, Clone)]
+#[group(skip)]
+pub struct Cmd {
+    #[command(flatten)]
+    pub config_locator: locator::Args,
```

Comment:
Do we need this now that it's a truly global arg?

Reply:
We do. We need to inject it, when that happens, then it can be used anywhere in the command. Just defining the switches as global doesn't automatically make them available.